### DISCUSSION TITLE
[Prettier Plugin]  VS code Prettier extension

### DISCUSSION THREAD

**Discussion by Nico-Mayer** [tailwindlabs/tailwindcss#12547] [Help]
Hello, I'm exploring the best approach to integrate this plugin seamlessly with the Prettier VSCode extension. My goal is to avoid installing Prettier individually for each project, and I prefer utilizing the settings configured within the VSCode plugin. Ideally, I'd like to skip having a .prettierrc file in each of my projects as well. What would be the recommended method to achieve this?

**Comment by nikolailehbrink** [comment]
Hi @Nico-Mayer, I understand you're looking to integrate the [Prettier plugin for TailwindCSS](https://github.com/tailwindlabs/prettier-plugin-tailwindcss) globally?! Is that correct? It's important to note that Prettier and TailwindCSS don't have a direct connection. You can easily install the Prettier extension in VSCode and set it as the default formatter for your files, which should work without a specific Prettier configuration file. However, if you're aiming to use the TailwindCSS plugin mentioned without a config file, that won't be possible. This limitation stems from a change in Prettier's behavior: when Prettier updated from version 2 to 3, they removed the automatic loading of plugins. This was reportedly due to incompatibility issues with some package managers, as noted in this [discussion](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/210#issuecomment-1694905954).
